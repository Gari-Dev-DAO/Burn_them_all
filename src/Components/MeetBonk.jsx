

const MeetBonk = () => {
  return (
    <div className='meetBonk'>
    <img  src={require('../assests/Images/meet-bonk.png')} alt='meet-bonk' className='meetBonkImage'/>
    </div>
  )
}

export default MeetBonk