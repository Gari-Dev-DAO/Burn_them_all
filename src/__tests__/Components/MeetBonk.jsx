import { render } from "@testing-library/react";
import MeetBonk from "../../Components/MeetBonk";

describe("Bonk Title", () => {
  it("renders component", () => {
    const { container } = render(<MeetBonk />);
    expect(container).toMatchSnapshot();
  });
});