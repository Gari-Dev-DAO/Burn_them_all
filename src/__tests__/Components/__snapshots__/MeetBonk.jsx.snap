// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bonk Title renders component 1`] = `
<div>
  <div
    class="meetBonk"
  >
    <img
      alt="meet-bonk"
      class="meetBonkImage"
      src="meet-bonk.png"
    />
  </div>
</div>
`;
